packages/@aws-cdk/aws-ecs/test/images/tag-parameter-container-image.test.ts

```diff
@@ -19,6 +19,21 @@ nodeunitShim({
         SynthUtils.synthesize(stack);
       }, /TagParameterContainerImage must be used in a container definition when using tagParameterName/);
 
+      test.done();
+    },
```

Comment:
Can you add an empty line after this line?

```suggestion
    },

```